website/docs/events/m3-slides/.slides/os-materials.md

```diff
@@ -1,24 +1,151 @@
 ---
 
-## OS materials
+## Operating Systems Content & Resources
 
----
+- [Presentation](https://open-education-hub.github.io/operating-systems/)
+- [Repository](https://github.com/open-education-hub/operating-systems/)
 
-### Lecture
+----
 
----
+### Lectures
+
+![Lecture](media/screenshot-os-lecture.png)
+
+----
+
+### Lectures
+
+![Lecture](media/screenshot-os-repository-lecture.png)
+
+----
+
+### Lectures
+
+- [(GitHub Flavored) Markdown](https://github.github.com/gfm/)
+- [reveal-md](https://github.com/webpro/reveal-md)
+- [Draw.io](https://drawio-app.com) + [ffmpeg](https://ffmpeg.org/)
+- Quizzes
+- Demos
+- Mardown linter, [Super-Linter](https://github.com/github/super-linter)
+- CC BY-SA 4.0
+
+----
 
 ### Practical Sessions
 
----
+![Lab](media/screenshot-os-lab.png)
+
+----
+
+### Practical Sessions
+
+![Lab](media/screenshot-os-repository-lab.png)
+
+----
+
+### Practical Sessions
+
+- [(GitHub Flavored) Markdown](https://github.github.com/gfm/)
+- Source code: solutions, generated skeletons, tests / checkers
+- Quizzes
+- [Super-Linter](https://github.com/github/super-linter)
+- BSD-3-Clause, CC BY-SA 4.0
+
+----
 
 ### Quizzes
 
----
+![Quizzes](media/screenshot-os-quiz.png)
+
+----
+
+### Quizzes
+
+- [(GitHub Flavored) Markdown](https://github.github.com/gfm/)
+- Integration with quiz management engine (Moodle, TutorWeb etc.)
+- Spread in Lectures and Practical Sessions
+- Public and private repositories
+
+----
 
 ### Demos
 
----
+![Demo](media/screenshot-os-demo.png)
+
+----
+
+### Demos
+
+- Source code
+- Used in Lectures
+- [Super-Linter](https://github.com/github/super-linter)
+
+----
+
+### Assignments
+
+![Assignment](media/screenshot-os-assignment.png)
+
+----
+
+### Assignments
+
+![Assignments](media/screenshot-os-gitlab-submissions.png)
+
+----
+
+### Assignments
+
+![Assignments - Repository View](media/screenshot-os-repository-assignment.png)
+
+----
+
+### Assignments
+
+- Source code: solutions, generated skeletons, tests / checkers
+- Public and private repositories
+- [Super-Linter](https://github.com/github/super-linter)
+- Submission via [vmchecker-next](https://github.com/systems-cs-pub-ro/vmchecker-next)
+
+----
+
+### Compliance Checking Infrastructure
+
+![Compliance Checking Infrastructure](media/screenshot-os-linter.png)
+
+----
+
+### Compliance Checking Infrastructure
+
+- [GitHub Action workflows](https://docs.github.com/en/actions/using-workflows)
+- [Super-Linter](https://github.com/github/super-linter)
+- [Checkpatch](https://docs.kernel.org/dev-tools/checkpatch.html)
+
+----
+
+### Deployment Infrastructure
+
+![Deployment Infrastructure](media/screenshot-os-site.png)
```

Comment:
Our landing page is currently the default one. Choose another page until https://github.com/open-education-hub/operating-systems/issues/106 is fixed.